### ISSUE TITLE
A11y violations in the user attribute search form

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#24528]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

There are a few critical and serious a11y violations in the user attribute search form that need to be fixed.

### Version

22.0.5

### Expected behavior

Violations are fixed.

### Actual behavior

Violations are detected.

### How to Reproduce?

Navigate to `Users`, select `Attribute search` and `Select attributes`. Once the form opens, run deque scan.
